nipype/algorithms/tests/test_confounds.py

```diff
@@ -43,7 +43,7 @@ def test_dvars(tmpdir):
                          in_mask=example_data('ds003_sub-01_mc_brainmask.nii.gz'),
                          save_all=True,
                          intensity_normalization=0)
-    os.chdir(str(tmpdir))
+    os.chdir(tmpdir.strpath)
```

Comment:
tmpdir is a `py.path.local` object which has a lot of methods:

http://py.readthedocs.io/en/latest/path.html#py-path-local-local-file-system-path

Reply:
true, you can do `tmpdir.chdir()` here :+1: